### PR TITLE
Update oneVPL patch 0076/0077 and delete 0078

### PR DESCRIPTION
**Pull request by xhaihao**
Update some strings and set the default device to d3d11va on Windows

Squash 0076 and 0078 into one commit because the support for D3D11 has
been upstreamed.

Re-create all patches after 0077

Signed-off-by: Haihao Xiang <haihao.xiang@intel.com>

### CODE REVIEW

**Review by uartie — APPROVED** [review]
LGTM

File: patches/0076-qsv-use-a-new-method-to-create-mfx-session-when-usin.patch [hunk 1]
```diff
@@ -1007,6 +1013,44 @@ index a8bc8c8143..d3c061b739 100644
      return ret;
  }
  
+@@ -1523,6 +1746,13 @@ static int qsv_device_create(AVHWDeviceContext *ctx, const char *device,
+         }
+     } else if (CONFIG_VAAPI) {
+         child_device_type = AV_HWDEVICE_TYPE_VAAPI;
++#if QSV_ONEVPL
++    } else if (CONFIG_D3D11VA) {  // Use D3D11 by default if d3d11va is enabled
++
```
Comment (galinart) [inline patches/0076-qsv-use-a-new-method-to-create-mfx-session-when-usin.patch:1022]: av_log(NULL, AV_LOG_WARNING,
                "WARNING: defaulting child_device_type to CONFIG_D3D11VA for oneVPL"
                "Please explicitly set device type via \"-init_hw_device\" option. if needed\n");

File: patches/0076-qsv-use-a-new-method-to-create-mfx-session-when-usin.patch [hunk 2]
```diff
@@ -1007,6 +1013,44 @@ index a8bc8c8143..d3c061b739 100644
...
+@@ -1557,6 +1788,15 @@ static int qsv_device_create(AVHWDeviceContext *ctx, const char *device,
+ #endif
+ #if CONFIG_DXVA2
+     case AV_HWDEVICE_TYPE_DXVA2:
++#if QSV_ONEVPL
++        {
++            av_log(NULL, AV_LOG_WARNING,
++                   "WARNING: d3d11va is not available or child device type is "
++                   "set to dxva2 explicitly for oneVPL code. Note oneVPL "
++                   "runtimes support d3d11va only on Microsoft Windows. Please "
```
Comment (galinart) [inline patches/0076-qsv-use-a-new-method-to-create-mfx-session-when-usin.patch:1047]: dxva2 is supported only in compatibility mode and new oneVPL features may not be supported

File: patches/0076-qsv-use-a-new-method-to-create-mfx-session-when-usin.patch (hunk 1 above)
Comment (xhaihao) [reply patches/0076-qsv-use-a-new-method-to-create-mfx-session-when-usin.patch]: Thanks, I updated the PR.

**Pull request merged**